docs/migration_guide.md

```diff
@@ -14,61 +14,83 @@ Since the migration might alter the database schema and the settings of `harbor.
 - With the introduction of storage and artifact quotas in version 1.9.0, migration from 1.7.x and 1.8.x might take a few minutes. This is because the `core` walks through all blobs in the registry and populates the database with information about the layers and artifacts in projects.
 - With the introduction of storage and artifact quotas in version 1.9.0, replication between version 1.9.0 and a previous version of Harbor does not work. You must upgrade all Harbor nodes to 1.9.0 if you have configured replication between them.
 
+
+
+
+
 ## Upgrading Harbor and Migrating Data
 
-1. Log in to the host that Harbor runs on, stop and remove existing Harbor instance if it is still running:
+#### Shutdown Harbor Service
 
-    ```sh
-    cd harbor
-    docker-compose down
-    ```
+Log in to the host that Harbor runs on, stop and remove existing Harbor instance if it is still running:
```

Comment:
remove as in delete?

Reply:
Log in to the host on which Harbor is running. Stop the existing Harbor instance if it is still running, and delete it: